tuf/api/metadata.py

```diff
@@ -966,7 +987,11 @@ class Timestamp(Signed):
         version: The metadata version number.
         spec_version: The supported TUF specification version number.
         expires: The metadata expiry date.
-        unrecognized_fields: Dictionary of all unrecognized fields.
+        unrecognized_fields: A Dictionary of all unrecognized attribute-value
+            fields. It guarantees that input containing unknown fields will
+            not break serialization from and deserialisation into a Metadta
+            object and thus aligns with the TUF specification implication that
+            unrecognized attribute-value fields should be supported
         snapshot_meta: Meta information for snapshot metadata.
```

Comment:
This is one I wondered whether to edit. It relates to obvious and unnecessary information, but on the other side it's simple and correct. I wondered what's your opinion about explaining why it's available in the API - do you find such clarification good or rather unnecessary?

Reply:
Yeah documenting meta in a comprehensive manner requires a lot of specification context so maybe we leave it as is... The MetaFile type does clearly document its content after all.